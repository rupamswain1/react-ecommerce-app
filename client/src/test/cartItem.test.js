import {shallow, mount,render} from 'enzyme';
import React from 'react';
import CollectionsPreview from '../components/collections-preview/collections-preview.component';
import {configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
//import Adapter from 'enzyme-adapter-react-16';

configure({adapter:new Adapter()});

it('expect to render StripChekcOut',()=>{
    const mockRobots=[{
        title:'hats',
        items:[]
    }]
    expect(shallow(<CollectionsPreview title={mockRobots[0].title} items={mockRobots[0].items}/>)).toMatchSnapshot('1')
});


