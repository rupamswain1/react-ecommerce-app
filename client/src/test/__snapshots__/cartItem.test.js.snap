// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render StripChekcOut: 1 1`] = `ShallowWrapper {}`;
